scripts/run-swiftlint.sh

```diff
@@ -24,7 +24,7 @@ SCRIPTS_DIR="$REPO_ROOT/scripts"
 SWIFTLINT="$SCRIPTS_DIR/.build/artifacts/scripts/SwiftLintBinary/SwiftLintBinary.artifactbundle/swiftlint-0.53.0-macos/bin/swiftlint"
 
 if [ -z "${CI-}" ]; then
-    xcrun --sdk macosx swift package --sdk macos --package-path "$SCRIPTS_DIR" resolve
+    xcrun --sdk macosx swift package --package-path "$SCRIPTS_DIR" resolve
```

Comment:
The redundant `--sdk` I forgot to fix in PR https://github.com/wireapp/wire-ios/pull/653